### PR TITLE
Function for import funds

### PR DESCRIPTION
**Pull request by akp111**

### CODE REVIEW

File: src/constants/screen.ts [hunk 1]
```diff
@@ -16,7 +16,8 @@ export const taskConstants: ScreenConstantsInterface = {
   'Delegate to a validator node': "delegate",
   'Get Mirror fund details': 'mirror',
   'Claim Rewards': "rewards",
-  'Withdraw funds': "withdrawal"
+  'Withdraw funds': "withdrawal",
+  'Import Funds (incase moving assests from C-Chain to P-Chain fails)': 'import',
```
Comment (avisingh2) [inline src/constants/screen.ts]: it needs to be rephrase since it is also applicable for p to c chain

File: src/interactive/cli.ts [hunk 2]
```diff
@@ -361,8 +361,76 @@ export async function interactiveCli(baseargv: string[]) {
...
+      trxType = 'CP'
+    if (importDestChain.type == 'C')
+      trxType = 'PC'
+
+    if (walletProperties.wallet == Object.keys(walletConstants)[0] && fileExists("ctx.json")) {
+      const { network: ctxNetwork, derivationPath: ctxDerivationPath } = readInfoFromCtx("ctx.json")
+      if (ctxNetwork && ctxDerivationPath) {
+        const argsImport = [...baseargv.slice(0, 2), "transaction", `import${trxType}`, "--blind", "true", "--derivation-path", ctxDerivationPath, `--network=${ctxNetwork}`, "--ledger"]
+        // ask for fees if its importTxPC
+        if (importDestChain.type == 'C') {
```
Comment (avisingh2) [inline src/interactive/cli.ts:377]: what about the case of p import transaction ? how you handle if user pass lowercase p or c

File: src/constants/screen.ts [hunk 3]
```diff
@@ -15,8 +15,9 @@ export const taskConstants: ScreenConstantsInterface = {
   'Add a validator node': 'stake',
   'Delegate to a validator node': 'delegate',
   'Get Mirror fund details': 'mirror',
-  'Claim Rewards': 'rewards',
-  'Withdraw funds': 'withdrawal',
+  'Claim Rewards': "rewards",
+  'Withdraw funds': "withdrawal",
```
Comment (avisingh2) [inline src/constants/screen.ts]: put single qoutes

File: src/constants/screen.ts (hunk 1 above)
Comment (avisingh2) [reply src/constants/screen.ts]: it didn’t change. you overwrite the change with latest commit.

File: src/interactive/cli.ts (hunk 2 above)
Comment (akp111) [reply src/interactive/cli.ts:377]: Its a drop down not something user manually imports

**Review by avisingh2 — APPROVED** [review]

**Pull request merged**